src/templates/pycontw-2020/ext/live.html

```diff
@@ -0,0 +1,56 @@
+{% extends 'contents/_base.html' %}
+
+{% load i18n compress %}
+
+{% block title %}{% trans 'Live broadcast' %}{% endblock %}
+
+{% block body_class %}{{ block.super }} live-page{% endblock body_class %}
+
+{% compress js %}
+{% block scripts %}
+{{ block.super }}
+<script>
+var next = document.querySelector("[name='next']");
+var params = new URLSearchParams();
+params.append('token', '{{ token }}');
+next.value += `?${params.toString()}`;
```

Comment:
What happens when the element `next` already has an existing value? Is this a possible situation?
Would it be like `?token=XXX&other_param=YYY?token=ZZZ&other_param=YYY`?

Reply:
Nope, I don't think any other query parameters would exist, particular in this page. This is why the code is added only in this page particularly.